.github/workflows/ci.yml

```diff
@@ -63,16 +81,25 @@ jobs:
           - ember-beta
           - ember-canary
           - ember-classic
+          - embroider-safe
```

Comment:
Should we keep embroider safe?

Reply:
You think we should remove it?

Reply:
Nevermind I always get confused between app and addon, we should keep it!